src/main/java/racingcar/InputViewCarName.java

```diff
@@ -0,0 +1,18 @@
+package racingcar;
+
+import java.util.Scanner;
+
+public class InputViewCarName<T> extends InputView<T>{
+
+    public InputViewCarName(String text) {
+        super(text);
+    }
+
+    @Override
+    T getInput() {
+        super.printText();
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        return (T) input.split(",");
+    }
+}
```

Comment:
잘 바꿔 주셨어요! `InputViewCarName` 는 구체 클래스이므로 타입을 지정해 줄 수 있을것 같아요.
```suggestion
public class InputViewCarName extends InputView<String[]>{

    public InputViewCarName(String text) {
        super(text);
    }

    @Override
    String[] getInput() {
        super.printText();
        Scanner sc = new Scanner(System.in);
        String input = sc.nextLine();
        return input.split(",");
    }
}
```